src/include/executor/tuptable.h

```diff
@@ -140,6 +148,8 @@ typedef struct TupleTableSlot
 
     /* System attributes */
     Oid         tts_tableOid;
+
+    SlotDeformTupleCodeGenInfo slot_deform_tuple_gen_info;
```

Comment:
I think that we need to have a pointer to another struct that has pointers to all xxxxCodeGenInfo structs. In that case, we will only have a pointer (e.g., CodeGenFunctions *) in TupleTableSlot.

```
typedef struct CodeGenFunctions{
     SlotDeformTupleCodeGenInfo *slot_deform_tuple_gen_info;
} CodeGenFunctions;

struct TupleTableSlot{
    ....
    ....
    CodeGenFunctions *codegened_funcs;
}
```